chapters/hpmor-chapter-001.tex

```diff
@@ -1,176 +1,175 @@
+Usually Hari was pretty good at answering that question, but in this particular case, he had no \emph{clue} what his brain was thinking.
 
-Harry mentally shrugged. A flat metal plate on a door affords pushing, and a handle on a door affords pulling, and the thing to do with a testable hypothesis is to go and test it.
+Hari mentally shrugged. A flat metal plate on a door affords pushing, and a handle on a door affords pulling, and the thing to do with a testable hypothesis is to go and test it.
 
 He took a piece of lined paper from his desk, and started writing.
 
 \begin{writtenNote}
 \letterAddress{Dear Deputy Headmistress}
 \end{writtenNote}
 
-Harry paused, reflecting; then discarded the paper for another, tapping another millimetre of graphite from his mechanical pencil. This called for careful calligraphy.
+Hari paused, reflecting; then discarded the paper for another, tapping another millimetre of graphite from his mechanical pencil. This called for careful calligraphy.
 
 \begin{writtenNote}
-\letterAddress{Dear Deputy Headmistress Minerva McGonagall,}
+\letterAddress{Dear Deputy Headmistress Minerva Mikgonagol,}
 
 \letterAddress{Or Whomsoever It May Concern:}
 
-I recently received your letter of acceptance to Hogwarts, addressed to Mr~H.~Potter. You may not be aware that my genetic parents, James Potter and Lily Potter (formerly Lily Evans) are dead. I was adopted by Lily’s sister, Petunia Evans-Verres, and her husband, Michael Verres-Evans.
+I recently received your letter of acceptance to Hogwarts, addressed to Mr~H.~Potr. You may not be aware that my genetic parents, Ĝajmz Potr and Lili Potr (formerly Lili Evanz) are dead. I was adopted by Lili’s sister, Petunja Evanz-Veres, and her husband, Mikael Veres-Evanz.
 
-I am extremely interested in attending Hogwarts, conditional on such a place actually existing. Only my mother Petunia says she knows about magic, and she can’t use it herself. My father is highly sceptical. I myself am uncertain. I also don’t know where to obtain any of the books or equipment listed in your acceptance letter.
+I am extremely interested in attending Hogwarts, conditional on such a place actually existing. Only my mother Petunja says she knows about magic, and she can’t use it herself. My father is highly sceptical. I myself am uncertain. I also don’t know where to obtain any of the books or equipment listed in your acceptance letter.
 
-Mother mentioned that you sent a Hogwarts representative to Lily Potter (then Lily Evans) in order to demonstrate to her family that magic was real, and, I presume, help Lily obtain her school materials. If you could do this for my own family it would be extremely helpful.
+Mother mentioned that you sent a Hogwarts representative to Lili Potr (then Lili Evanz) in order to demonstrate to her family that magic was real, and, I presume, help Lili obtain her school materials. If you could do this for my own family it would be extremely helpful.
```

Comment:
```suggestion
Mother mentioned that you sent a Hogŭartso representative to Lili Potr (then Lili Evanz) in order to demonstrate to her family that magic was real, and, I presume, help Lili obtain her school materials. If you could do this for my own family it would be extremely helpful.
```